route/build/src/write_restart_pio.f90

```diff
@@ -336,6 +320,9 @@ SUBROUTINE define_state_nc(fname,           &  ! input: filename
  call def_var(pioFileDescState, 'restart_time', ncd_float, ierr, cmessage, vdesc='resatart time', vunit=trim(time_units), vcal=calendar)
  if(ierr/=0)then; message=trim(message)//trim(cmessage); return; endif
 
+ call def_var(pioFileDescState, 'history_time', ncd_float, ierr, cmessage, vdesc='history time', vunit=trim(time_units), vcal=calendar)
```

Comment:
I'd make these two double for restarts. It probably won't matter, but restarts you want to ensure are exact so output in the native precision.